README.md

```diff
@@ -166,6 +166,11 @@ $ ./qltool run -f examples/rootfs/x8664_linux/bin/x8664_hello --gdb 127.0.0.1:99
 
 See  https://docs.qiling.io/  for more details
 
+With code coverage collection (UEFI only for now):
+
+```
+$ ./qltool run -f TcgDxe.efi --rootfs . --coverage-format drcov --coverage-file TcgDxe.cov
```

Comment:
is should be ,./qltool run -f examples/rootfs/xxxx

Reply:
Fixed.